### PR TITLE
Adds two new beers: lager and stout 

### PR DESCRIPTION
**Pull request by ChocoboLord**
## What Does This PR Do
This PR allows the bartender to serve two brand new beers, lager and stout. Functionally the same as most other beers, except stout can replenish blood thanks to its high iron content (which is a real thing!)

## Why It's Good For The Game
The bartender could do with more options for drinks so that people don't just order gargle blasters all shift, so hopefully this'll help add a bit more spice to bar RPs. And anyway, who doesn't want more excuses to get drunk on duty?
Also, the concept of saving someone from exsanguination with beer fits with the humour of the server.

## Images of changes
stoutnlager
starlink
stout

## Testing
Spawned in, went to bar, poured myself two pints and drank them both. To test the iron-like effects of the stout, I battered Pun Pun then fed him some stout after he'd bled a bit. Worked perfectly.

## Changelog
:cl:
add: Added lager and stout to the booze dispensers
/:cl:

### CODE REVIEW

**Comment by ChocoboLord** [comment]
Someone on the discord pointed out letting it replenish as much blood as iron probably isn't a good idea, so I lowered the replenishment a bit

File: code/modules/reagents/chemistry/reagents/alcohol.dm [hunk 1]
```diff
@@ -1862,3 +1862,33 @@
...
+	drink_name = "Stout"
+	drink_desc = "A pitch black beer from Ireland, high in iron content"
+	taste_description = "the luck of the Irish"
+
+/datum/reagent/consumable/ethanol/stout/on_mob_life(mob/living/M) // Replenishes blood, seeing as there's iron in it
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(!(NO_BLOOD in H.dna.species.species_traits))
+			if(H.blood_volume < BLOOD_VOLUME_NORMAL)
+				H.blood_volume += 0.4
```
Comment (DGamerL) [inline code/modules/reagents/chemistry/reagents/alcohol.dm:1893]:
```suggestion
		if(!(NO_BLOOD in H.dna.species.species_traits) && (H.blood_volume < BLOOD_VOLUME_NORMAL))
			H.blood_volume += 0.4
```

**Comment by DGamerL** [comment]
image

What in the world

**Comment by ChocoboLord** [comment]
> image
> 
> What in the world

Wait what? I ADDED two sprites, how did THAT happen?

**Review by DGamerL — APPROVED** [review]
Code LGTM, just needs a sprite review

**Comment by ChocoboLord** [comment]
The suspense of not knowing whether my amateur sprite work is good enough is killing me

**Review by DGamerL — APPROVED** [review]
PR is now fully approved, just needs a second code review

**Review by Burzah — APPROVED** [review]

**Pull request merged**